lists/broken

```diff
@@ -32,6 +31,7 @@ binaryninja-python # discontinued, better delete
 binflow # abandonned since 2018, doesn't build, see #3748
 birp #  not working, unmaintained since 2018, python 2, see #4080
 climber # discontinued, python2, issue with python2-paramiko
+cmsscanner # #3519
```

Comment:
We should try to rebuild it now, with a newer version of ruby and securerandom

https://github.com/rubygems/rubygems/issues/5941#issuecomment-2355711125